src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const tableElem = document.querySelector('table');
+const tHead = tableElem.firstElementChild;
+const tBodyList = [...tHead.nextElementSibling.children];
+const tFoot = tableElem.lastElementChild;
```

Comment:
instead of getting separate elements, get all of the `tr` elements and loop through them
```javascript
const rows = document.querySelectorAll('tr');
```